schemas/triniti/news/mixin/search-articles-request/latest.xml

```diff
@@ -9,6 +9,9 @@
       <field name="is_unlisted" type="trinary">
         <default>2</default>
       </field>
+      <field name="is_locked" type="trinary">
+        <default>2</default>
```

Comment:
@hunyoboy remove the default, in this case we want "0" aka "unset/unknown" to be used.